CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ find_package(ZLIB REQUIRED)
 find_package(MiniZip 1 QUIET) # version range (0...<2.0.0) requires CMake>=3.19
 if(UNIX AND NOT APPLE)
     find_package(PkgConfig)
-    pkg_check_modules(MINIZIP minizip IMPORTED_TARGET minizip<2.0.0)
+    pkg_check_modules(MINIZIP minizip IMPORTED_TARGET minizip<5.0.0)
```

Comment:
Why 5? do we need version restriction?

Reply:
It was there before, and I wasn't sure if it served some purpose, so I left it there

Reply:
I think it was there to avoid pickup minizip-ng. My question is why version 5 or can we get rid of this condition?

Reply:
From what I could test, version 3 is compatible after these two changes. I can test version 4 as well, but I cannot guarantee the upcoming ones.

Reply:
@metsma I've tested it on version 4 and it's building, so I would leave it this way. Can you merge this?

Reply:
Here is my build: https://copr.fedorainfracloud.org/coprs/ljavorsk/minizip-ng/build/6880707/

Reply:
Why not remove then the <5 condition? If new release is with major version number 5 then we need change code again